lib/index.js

```diff
@@ -393,7 +451,7 @@ function morphdom(fromNode, toNode, options) {
             if (toNodeType === 1) {
                 if (fromNode.tagName !== toNode.tagName) {
```

Comment:
Its not enough to only compare tagName when you want to be namespace aware.
Imagine you create a custom htmlelement with tagname : path
And you would will compare it to SVGPathElement (path). Then this expression will return true but should return false.
So also namespaceURI should be checked.